internal/display/apis.go

```diff
@@ -46,16 +51,18 @@ type apiTableView struct {
 	Name       string
 	Identifier string
 	Scopes     int
+	TokenLifetime int
+	AllowOfflineAccess bool
 
 	raw interface{}
 }
 
 func (v *apiTableView) AsTableHeader() []string {
-	return []string{"ID", "Name", "Identifier", "Scopes"}
+	return []string{"ID", "Name", "Identifier", "Scopes", "TokenLifetime", "AllowOfflineAccess"}
```

Comment:
Let's keep these new fields off the table view, as there's already enough info in there.